compiler/vmgen.nim

```diff
@@ -29,7 +29,7 @@
 
 import
   strutils, ast, astalgo, types, msgs, renderer, vmdef,
-  trees, intsets, magicsys, options, lowerings, lineinfos, transf
+  trees, intsets, magicsys, options, lowerings, lineinfos, transf, asciitables
```

Comment:
Make this import dependent on ``nimVMDebug`` I don't want to pay for it when I don't debug the VM (which is "almost always").

Reply:
done